src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/ComEventsHelperTests.cs

```diff
@@ -11,10 +11,18 @@ public class ComEventsHelperTests
     {
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
         public void Combine_NullRcwWindows_ThrowsArgumentNullException()
         {
             AssertExtensions.Throws<ArgumentNullException>(null, () => ComEventsHelper.Combine(null, Guid.Empty, 1, null));
         }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapAot, "Throws PlatformNotSupportedException in UapAot")]
+        public void Combine_NullRcwUapAot_PlatformNotSupportedException()
```

Comment:
Why not make coreclr and uapaot have the same behaviour for null argument to combine? Coreclr throws a PNSE after validating arguments unconditionally so maybe this would make sense for uap too